<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FridaGuiTool</class>
 <widget class="QMainWindow" name="FridaGuiTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frida GUI Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="PythonScriptBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>740</y>
      <width>680</width>
      <height>130</height>
     </rect>
    </property>
    <property name="title">
     <string>2. Python Script</string>
    </property>
    <widget class="QListWidget" name="PythonScriptList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>660</width>
       <height>30</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
    <widget class="QPushButton" name="PythonScriptsButton">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>55</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Hooking</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="PythonScriptsLog">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>660</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="PythonScriptLoad">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Select PythonScript</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="Information">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>680</width>
      <height>60</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Package</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Memory Base</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>IDA Base</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>PID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>OS</string>
     </property>
    </column>
    <item row="0" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="5">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Path</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Path">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>650</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="TabManager">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>125</y>
      <width>685</width>
      <height>610</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Start</string>
     </attribute>
     <widget class="QTextBrowser" name="ScriptLog">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>660</width>
        <height>570</height>
       </rect>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::NoWrap</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>105</y>
      <width>270</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1. Set Options (Hooking Scripts / ETC Options)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetOptionsBtn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>105</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Options</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartHooking">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>105</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Hooking</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="analysisMenu"/>
   <addaction name="separator"/>
   <addaction name="configMenu"/>
   <addaction name="separator"/>
  </widget>
  <action name="analysisMenu">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Analysis</string>
   </property>
  </action>
  <action name="configMenu">
   <property name="text">
    <string>Options</string>
   </property>
   <property name="toolTip">
    <string>Options</string>
   </property>
  </action>
  <action name="SelAttachDevice">
   <property name="text">
    <string>Device 변경</string>
   </property>
   <property name="toolTip">
    <string>Device 변경</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
